.travis.yml

```diff
@@ -14,20 +14,13 @@ services:
 env:
   matrix:
     - MODE=source-checks
-    - MODE=test-tools SBT_VERSION=0.13.17
-    - MODE=test-tools SBT_VERSION=1.2.6
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=boehm SBT_VERSION=0.13.17 TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=boehm SBT_VERSION=1.2.6 TEST_COMMAND="sbtScalaNative/scripted"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=immix SBT_VERSION=0.13.17 TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=immix SBT_VERSION=1.2.6 TEST_COMMAND="sbtScalaNative/scripted"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=commix SBT_VERSION=0.13.17 TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=debug SCALANATIVE_GC=commix SBT_VERSION=1.2.6 TEST_COMMAND="sbtScalaNative/scripted"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=boehm SBT_VERSION=0.13.17 TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=boehm SBT_VERSION=1.2.6 TEST_COMMAND="sbtScalaNative/scripted"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=immix SBT_VERSION=0.13.17 TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=immix SBT_VERSION=1.2.6 TEST_COMMAND="sbtScalaNative/scripted"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=commix SBT_VERSION=0.13.17 TEST_COMMAND="test-runtime"
-    - MODE=test-runtime TARGET_DOCKER_PLATFORM=library SCALANATIVE_MODE=release-fast SCALANATIVE_GC=commix SBT_VERSION=1.2.6 TEST_COMMAND="sbtScalaNative/scripted"
+    - MODE=test-tools SBT_VERSION=1.3.7
```

Comment:
`SBT_VERSION` should match the value of `sbt10Version` in `build.sbt`, not the `sbt.version` of `project/build.properties`. So this should be `1.0.4`.

Same in all the lines below.

Reply:
Thanks for the review.

So we run the following directly in Travis outside of Docker:
`sbt "-no-colors" "-J-Xmx3G" "^^ $SBT_VERSION" nscplugin/publishLocal sbtScalaNative/publishLocal test-tools`
So I guess the idea is we test the tools with the minimum `sbt` and launcher version.

Then we test the runtimes using the latest `sbt` so hopefully it is better and faster?

Reply:
I looked in the CI logs and it is using the `sbt` launcher version `1.3.7` but then using `1.0.4` later in the build `[info] Setting` `` `sbtVersion in pluginCrossBuild` `` `to 1.0.4`. Also, it seems that some deprecations may not be able to be removed until the `sbt` minimum version is updated based on the warnings. Hopefully, if someone if working on the tools then some of this code can be updated to remove deprecations.

I guess too if someone is using an older launcher and it fails then it is ok for them too update the launcher. They can update `sbt` in their build on a different schedule.

Reply:
It should be set to 1.0.4 for *all* builds. This enjoyment variable only affects the version of sbt that the sbt plugin is built for. The build itself will always use the version of sbt that is defined in `build.properties`.

In fact, since we now only support sbt 1.x for the sbt plugin, this environment variable is redundant. It would be better to get rid of it entirely, and get rid of the `^^ ...` commands. All of that is redundant now.

Reply:
@sjrd The sbt slash syntax was not offered until 1.1.0 and now the scripted tests are running in 1.0.4 so this doesn't work. The idea was to support the latest in each series, 1.0.4, 1.1.6, 1.2.8 and 1.3.x. Maybe we should bump the minimum version to 1.1.6? Otherwise we need change the 3 scripted tests to use the non-slash syntax. I tested the 1.1.6 idea and that works.

Reply:
I pushed the `1.1.6` change but can go the other way if desired.